src/components/Card.tsx

```diff
@@ -49,13 +50,15 @@ const cornerSizeToBorderRadius: {
 }
 
 const Card = forwardRef<HTMLDivElement, CardProps>(({
-  cornerSize: size = 'large', hue = 'default', selected = false, clickable = false, ...props
+  cornerSize: size = 'large', hue = 'default', selected = false, clickable = false, form = false, ...props
 }, ref) => (
   <Div
     ref={ref}
     border={`1px solid ${hueToBorderColor[hue]}`}
     borderRadius={cornerSizeToBorderRadius[size]}
     backgroundColor={selected ? hueToSelectedBGColor[hue] : hueToBGColor[hue]}
+    paddingVertical={form ? 'xlarge' : null}
+    maxWidth={form ? '608px' : null}
```

Comment:
Isn't it too small? Looking at [`OpenID Connect` form design](https://www.figma.com/file/kZ69OGXTQe3Gu0wzGdDyoO/Marketplace-Tab?node-id=909%3A27997) card has 832px width. Where it would be used?

Reply:
If you take a look at https://github.com/pluralsh/plural/pull/393 cards seem to already have 832px width, but not sure if that's the place where it would be used.

Reply:
https://linear.app/pluralsh/issue/ENG-311/repository-refinement

Issue says to use max width of 608px

Reply:
I believe this is about forms and now they have 608px but the card has 832px because of paddings. Let's ask design team to confirm.

Reply:
Ye, you might be right. Let us confirm with the design team.

Reply:
As discussed, let's also clarify if `ContentCard` should not be used instead. Then only max width seems to be missing.